src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -26,8 +26,15 @@ public ListCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_DEPARTMENT);
+
         if (argMultimap.getValue(PREFIX_DEPARTMENT).isEmpty()) {
-            return new ListCommand();
+            try {
+                String secondArg = args.split(" ")[1];
```

Comment:
What does this try catch block do?

Reply:
When I split the string, if there are no parameters after "list" (array size of 1, index 1 is out of bounds) it will return a standard list command (lists all employees). This is needed because, unlike other commands, "list" alone is a valid command here. Only "list" + any other parameters besides the department are invalid.